test/lib/konnectors.spec.js

```diff
@@ -114,6 +82,18 @@ describe('konnectors lib', () => {
     })
   })
 
+  describe('isKonnectorUpdateNeededError', () => {
+    ;['TERMS_VERSION_MISMATCH'].forEach(errorType => {
```

Comment:
I see what you are targeting (future errors), but no need for loop here for now 😅